config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
+
+# Define an application-wide content security policy.
+# See the Securing Rails Applications Guide for more information:
+# https://guides.rubyonrails.org/security.html#content-security-policy-header
+
+# Rails.application.configure do
```

Comment:
Should some/any/all of these be uncommented?

Reply:
don't think so - early days